bindings/python/iota_sdk/types/block.py

```diff
@@ -18,11 +18,13 @@ class Block:
     Attributes:
         protocol_version: The protocol version with which this block was issued.
         parents: The parents of this block.
+        burned_mana: The amount of Mana the Account identified by the IssuerId is at most willing to burn for this block.
         payload: The optional payload of this block.
     """
 
     protocol_version: int
     parents: List[HexStr]
+    burned_mana: str
```

Comment:
Not a number?

Reply:
No, it gets serialized as string

Reply:
Yeah, but does that mean the field has to be a string in the type?

Reply:
Hm yeah if we add some custom dict/json methods then we should be able to do it, will have to investigate

Reply:
Do we really use strings for all u64 in python?

Reply:
Currently we do

Reply:
Dang, alright